training/DELIVERY.md

```diff
@@ -0,0 +1,248 @@
+
+As a general rule, for separation, two planes flying on the same SID and taking off one after the other will receive a different flight level clearance: the first will receive a maximum of FL280 (westbound) and FL270 (eastbound), if no other local procedures exist, and the second plane will be cleared to climb 2000 ft lower than the first. This rule is applied for each individual SID, i.e. for each traffic flow.
+
+A practical example: flights from LROP via SOKRU and via BUKEL. The first plane flying to SOKRU climbs to FL280, the second plane flying to SOKRU climbs to FL260, the third plane flying to SOKRU climbs to FL240 and so on. After that, the first plane flying towards BUKEL (being another flow) will climb to FL280, the second towards BUKEL will climb to FL260 and so on. 
+
+An already used flight level becomes available when the aircraft to which that level was assigned no longer appears in the Departure List, i.e. it has taken off.
+
+It should be noted that during the initial climb authorization (at clearance), the flight level or the altitude to which the pilot is cleared to climb must not exceed the flight level or the altitude that the pilot intends to maintain during the cruise phase of the flight.
+
+In phraseology, there are two phrases used to authorize an aircraft to climb to a certain altitude or flight level.
+
+The first, when there are no speed or altitude restrictions between two points on that SID:
+
+`CLIMB FL280`
+
+And the second, when there is at least one speed or altitude restriction in the SID:
+
+`CLIMB VIA SID FL280`
+
+### 2.5. Transponder Code
+
+The transponder is a device installed in the plane used to identify the aircraft and correlate the target with the flight plan. What you need to know about the transponder is that aircraft are required to use it in controlled airspace and that it can be operated in 3 modes.
+
+Functionality of the transponder: the pilot of the aircraft set a code in the transponder, a code that is transmitted to the radar, the radar identifies a position from where a signal is emitted and is transmitted to the CWP (controller work position), where ATC is tasked with identifying the aircraft that displays a certain code and link or correlate the target on the screen with the flight plan related to that aircraft. As a DEL controller, your job is not to identify the aircraft on the radar because you work in a non-radar environment, but only to instruct the pilot to set a certain transponder code.
+
+There are 4 operating modes of the transponder:
+
+1. STBY (stand-by): transponder is off; no signal is send to the radar
+2. ModeA: only the current position is send to the radar
+3. ModeC: the current position and altitude is send to the radar
+4. ModeS: is the most complicated mode, because in addition to ModeC's transmission capability, ModeS can also transmit other data about the aircraft, such as indicated air speed, mach number or fuel on board.
```

Comment:
```suggestion
4. ModeS: is the most complex mode, because in addition to ModeC's transmission capability, ModeS can also transmit other data about the aircraft, such as indicated air speed, mach number or fuel on board.
```